nebu/bakedpdf.py

```diff
@@ -0,0 +1,129 @@
+from lxml import etree
+
+from cnxepub.models import Binder, TranslucentBinder, Document
+from cnxepub.formatters import SingleHTMLFormatter
```

Comment:
Was sort of hoping we could avoid the epub models and formatters for this tool set, from the get go. However, I understand the desire to use existing code to get it done more quickly.  We'll just have to be really careful about the potential complexity this brings in.

Reply:
Right. However, I'm just an "implementer"—as you said 😄. Using existing models was a requirement set forth by Ed.

Reply:
No "just" in there - you should feel free to push back on requirements that impact implementation, regardless of the seniority of the one involved. We hire smart people for a reason. We're often guessing (though they _are_ highly educated guesses :-) ) and all know that the real constraints often don't show up until deep into the implementation. Sorry if you feel like you're getting trapped in the middle.

Reply:
We could skip cnx-epub models and SingleHTMLFormatter but I didn't feel we had enough discussion about the alternatives.  Single HTML is what cnx-recipes uses in the scripts for baking so that's what we're using here too.  It is ok to change the format and implementation if we have time now or do it after the March deadline.

Reply:
Agree that using them now is the best starting place, since that's what _current_ recipe/baking does, so that makes this tool immediately useful for textbook devs, but we do want to keep in mind replacing it.